Source/Engine/Core/Config/LayersAndTagsSettings.cs

```diff
@@ -30,6 +31,29 @@ public static string[] GetCurrentLayers()
             return GetCurrentLayers(out int _);
         }
 
+        /// <summary>
+        /// The layers names.
+        /// </summary>
+        [EditorOrder(10), EditorDisplay("Terrain Layers", EditorDisplayAttribute.InlineStyle), Collection(CanResize = false, Display = CollectionAttribute.DisplayType.Inline)]
+        public string[] TerrainLayers = Enumerable.Repeat(string.Empty, 8).ToArray();
```

Comment:
Done in https://github.com/FlaxEngine/FlaxEngine/pull/3021/commits/75d4a110de45b5494fd7ee3c1f49597797dd5605.

I also had to remove the `ToArray()`, which leads to the Terrain layer names being populated with `Layer {index}` by default:
![image](https://github.com/user-attachments/assets/0f5f1a90-c94e-4611-9fa1-57fb2c63f04a)
Before it looked like this:
![image](https://github.com/user-attachments/assets/87f877b9-5f96-45e4-979a-c02293b5d653)

The new behavior might actually make more sense because there will always be 8 terrain layers, so they all need names.

However if that's a problem I can try to find a solution.